CodeIgniter.gitignore

```diff
@@ -1,6 +1,13 @@
+.DS_Store
```

Comment:
Can you drop this line, as it's an OS-specific file we're tracking in `Global/macOS.gitignore`?